pandas/core/groupby/groupby.py

```diff
@@ -1266,8 +1266,40 @@ def count(self):
     def mean(self, *args, **kwargs):
         """
         Compute mean of groups, excluding missing values
+        
+        Example of groupby one column:
+        ------------------------------
+        >>> df = pd.DataFrame({'A': [1, 1, 2, 1, 2],
+                               'B': [np.nan, 2, 3, 4, 5]}, columns=['A', 'B'])
+        >>> g = df.groupby('A')['B'].mean()
+        >>> g
+        A
+        1    3.0
+        2    4.0
 
```

Comment:
You may want to put the Example at the end, see https://python-sprints.github.io/pandas/guide/pandas_docstring.html